lib/calendarium-romanum/cli.rb

```diff
@@ -115,5 +107,25 @@ def die!(message, code=1)
       STDERR.puts message
       exit code
     end
+
+    private
+
+    def print_single_date(calendar, date, sanctorale)
```

Comment:
argument `sanctorale` is unused, remove it

Reply:
Done. Just posting it here as a checklist for myself.